src/Components/AlbumList.tsx

```diff
@@ -1,16 +1,18 @@
-import './AlbumCardList.css';
+import './AlbumList.css';
 
 import {Album} from '../types/data';
 import AlbumCard from './AlbumCard';
 
 interface Props {
     data: Album[];
+    title: string;
 }
 
-const AlbumCardList = ({data}: Props) => {
+const AlbumList = ({data, title}: Props) => {
     return (
-        <>
-            <ul className='album'>
+        <div className='album'>
+            <h2>{title}</h2>
+            <ul className='albumlist'>
```

Comment:
Maybe we should have a check for if there are no results? i.e. data.length is 0. In this case we don't want an empty <ul></ul>